java/README.md

```diff
@@ -1,19 +1,68 @@
 Welcome To The New Evolution In Sauce Labs Testing.
 The goal of Simple Sauce is to make your test automation and integration in Sauce Labs, simple :)
+Our fluent API helps you to understand all of the possible browser/OS combinations at design time.
+You no longer need to read extensive docs, just let your IDE guide you 😉
 
-Start a test on all defaults:
+##Quick Start: a test on all defaults:
 
 ```WebDriver driver = new SauceSession().start();```
 ```driver.findElement("id").click()```
```

Comment:
I actually hate this line, since it's not even syntactically correct. I'll fix it up.